subworkflows/local/prepare_genome/main.nf

```diff
@@ -12,7 +12,6 @@ include { BWA_INDEX as BWAMEM1_INDEX                } from '../../../modules/nf-
 include { BWAMEM2_INDEX                             } from '../../../modules/nf-core/bwamem2/index/main'
 include { DRAGMAP_HASHTABLE                         } from '../../../modules/nf-core/dragmap/hashtable/main'
 include { GATK4_CREATESEQUENCEDICTIONARY            } from '../../../modules/nf-core/gatk4/createsequencedictionary/main'
-include { MSISENSORPRO_SCAN                         } from '../../../modules/nf-core/msisensorpro/scan/main'
```

Comment:
what is the reason to move this out?

Reply:
I moved it to main workflow file `main.nf` as it was unnecessarily generated if msisensorpro was not requested. As now I made `msisensorpro_scan` and `msisensorpro_baseline` optional params.